README.md

```diff
@@ -38,16 +37,15 @@ If you run into problems with go test caching, set the enviornment variable
 
 ## API
 
-The application exposes single endpoint, `/` - root. It requires two headers to
-be provided, in order for it to work as expected. These are:
+The application exposes single endpoint, `/metrics` - root. It requires two
+headers to be provided, in order for it to work as expected. These are:
 
 ### `Accept` header
 
 Which tells the application at what format would you like to receive the logs.
 Currently only two formats are being recognised:
 
-- JSON: `application/json`
-- Prometheus: `application/vnd.google.protobuf`
+- Prometheus: `text/plain`
```

Comment:
Did you intend to remove JSON here? The line above still says that 2 formats are recognised.

Reply:
No, I will amend this